login-form/src/redux/selectors/dashboardReselectors.js

```diff
@@ -0,0 +1,36 @@
+import { createSelector } from "reselect";
+
+export const reselectEmail = createSelector(
+  (state) => state.getDashboard.email,
+  (email) => {
+    return email;
+  }
```

Comment:
```suggestion
  (email) => email
```

you can simplify code like ☝️